### ISSUE TITLE
Generate tests only?

### ISSUE THREAD

**Issue by wizonesolutions** [hechoendrupal/DrupalConsole#902]
Is it possible to generate just tests? e.g. SimpleTest or PHPUnit scaffolding for a module.

**Comment by jmolivas** [comment]
@wizonesolutions: At this point this is not supported by the project, but will be great to add this feature, we can use this issue to discuss.

But you can take advantage and practice your Español by talking to @dmouse at Barcelona, during DrupalCon and have a conversation about how to achieve this, and also you can have a better understanding of the project if interested in contribute.

**Comment by darol100** [comment]
:+1:  For this feature that will be a very great for this project.

**Comment by enzolutions** [comment]
@wizonesolutions @darol100 do you have a use case about how this new command must to work?

**Comment by wizonesolutions** [comment]
@enzolutions No, not really. Maybe just a very simple scaffold that contains some stuff we can change to get going. Often the psychological barrier to _getting started_ with testing is too high.
